x-pack/plugins/security_solution/public/overview/components/host_overview/endpoint_overview/index.tsx

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiFlexItem, EuiHealth } from '@elastic/eui';
+import { getOr } from 'lodash/fp';
+import React from 'react';
+
+import { DescriptionList } from '../../../../../common/utility_types';
+import { getEmptyTagValue } from '../../../../common/components/empty_value';
+import { DefaultFieldRenderer } from '../../../../timelines/components/field_renderers/field_renderers';
+import { EndpointFields, HostPolicyResponseActionStatus } from '../../../../graphql/types';
+import { DescriptionListStyled } from '../../../../common/components/page';
+
+import * as i18n from './translations';
+
+interface Props {
+  data: EndpointFields | null;
+}
+
+const getDescriptionList = (descriptionList: DescriptionList[], key: number) => (
+  <EuiFlexItem key={key}>
+    <DescriptionListStyled data-test-subj="endpoint-overview" listItems={descriptionList} />
+  </EuiFlexItem>
+);
+
+export const EndpointOverview = React.memo<Props>(({ data }) => {
+  const getDefaultRenderer = (fieldName: string, fieldData: EndpointFields, attrName: string) => (
+    <DefaultFieldRenderer
+      rowItems={[getOr('', fieldName, fieldData)]}
+      attrName={attrName}
+      idPrefix="endpoint-overview"
+    />
+  );
+  const descriptionLists: Readonly<DescriptionList[][]> = [
```

Comment:
can we use `useMemo`? might also be a good idea to get `getDefaultRenderer` around a `useCallback`

Reply:
done